ultra/utils/common.py

```diff
@@ -0,0 +1,303 @@
+import torch
+import numpy as np
+import random
+import shutil
+import os, cv2, dill
+from matplotlib import pyplot as plt
+from skimage.transform import resize
+import ultra.utils.geometry as geometry
+from scipy.spatial.distance import euclidean
+import math
+
+
+def rotate2d_vector(vectors, angle):
+    ae_cos = np.cos(angle)
+    ae_sin = np.sin(angle)
+    rot_matrix = np.array([[ae_cos, -ae_sin], [ae_sin, ae_cos]])
+    vectors_rotated = np.inner(vectors, rot_matrix)
+    return vectors_rotated
+
+
+def plot_experiment_data(experiment_path, save_dir):
+    # logs/experiment-2020.9.25-12:16:14/pkls/Train
```

Comment:
Could this be misleading?

Reply:
we used this module for plots in the paper, why is this misleading?

Reply:
Sorry it was just a nit, I should have been specific, I meant the comment.